LOAN/RealEstateLoans/HomeMortgageDisclosureActCoveredMortgages.rdf

```diff
@@ -220,7 +220,7 @@
 		<skos:definition>a category based on a person&apos;s physical characteristics, such as bone structure and skin, hair, or eye color</skos:definition>
```

Comment:
Minefield here but "eye color" is bizarre.
"The Census Bureau defines race as a person's self-identification with one or more social groups. An individual can report as White, Black or African American, Asian, American Indian and Alaska Native, Native Hawaiian and Other Pacific Islander, or some other race."

Reply:
@rivettp - see my prior comments - I have no idea what HMDA requires, so we need an SME